vita3k/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -242,7 +242,7 @@ EXPORT(int, sceAppUtilSaveDataDataSave, SceAppUtilSaveDataFileSlot *slot, SceApp
         }
     }
 
-    if (slot) {
+    if (slot->slotParam) {
```

Comment:
```suggestion
    if (slot && slot->slotParam) {
```
if slot can still be nullptr

Reply:
ah i see